ldsc/src/main/resources/partitionRegions.py

```diff
@@ -122,8 +120,9 @@ def main():
         df.dataset
     )
 
-    # output the regions partitioned for LDSC in BED format
-    df.write \
+    df.orderBy(['chromosome', 'start', 'end']) \
+        .coalesce(1) \
```

Comment:
This supplants the sort / coalesce in the merging stage such that now we can just pass dataset-level partitions directly into the merged folder and not resort and reformat unnecessarily